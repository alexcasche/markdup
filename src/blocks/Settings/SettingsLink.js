import { Link } from 'elements'
import { shade, tint } from 'polished'

const SettingsLink = Link.extend`
  width: 100%;
  display: flex;
  justify-content: space-between;
  align-items: center;
  color: ${props => props.theme.colors.grey};
  height: 50px;
  text-transform: capitalize;
  border-bottom: 1px solid ${props => shade(0.5, props.theme.colors.grey)};
  transition: color 0.15s ease-in-out;
  &:hover {
    color: ${props => tint(0.25, props.theme.colors.grey)};
    cursor: pointer;
  }
  &:last-of-type {
    border-bottom: none;
  }
`

export default SettingsLink
