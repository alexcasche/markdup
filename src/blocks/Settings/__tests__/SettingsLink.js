import SettingsLink from '../SettingsLink'

describe('SettingsLink Element', () => {
  it('renders correctly', () => {
    const tree = render(<SettingsLink>Link</SettingsLink>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
