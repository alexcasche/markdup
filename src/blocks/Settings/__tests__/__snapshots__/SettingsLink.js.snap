// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsLink Element renders correctly 1`] = `
.c0 {
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.15s ease-in-out;
  transition: all 0.15s ease-in-out;
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #ABADB3;
  height: 50px;
  text-transform: capitalize;
  border-bottom: 1px solid #555659;
  -webkit-transition: color 0.15s ease-in-out;
  transition: color 0.15s ease-in-out;
}

.c0:hover {
  cursor: pointer;
}

.c0:hover {
  color: #eaeaec;
  cursor: pointer;
}

.c0:last-of-type {
  border-bottom: none;
}

<a
  className="c0"
>
  Link
</a>
`;
